File: dodam-infra/src/main/java/b1nd/dodaminfra/api/dodamtoken/JwtProperties.java
package b1nd.dodaminfra.api.dodamtoken;

import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.context.annotation.Configuration;

@Getter
@Setter
@Configuration
@ConfigurationProperties("app.jwt")
class JwtProperties {

    private String issuer;
    private String tokenServer;

}

File: core/core-security/src/main/kotlin/com/b1nd/dodamdodam/core/security/jwt/JwtProperties.kt
package com.b1nd.dodamdodam.core.security.jwt

import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties(prefix = "jwt")
data class JwtProperties(
    val secret: String = "default-secret-key-for-development-only-please-change-in-production",
    val accessTokenExpiration: Long = 3600000, // 1 hour
    val refreshTokenExpiration: Long = 604800000 // 7 days
)
